buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -598,6 +598,42 @@
         "failOnStandardError": "false"
       }
     },
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Package - Debian 9",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "docker",
+        "arguments": "run --rm $(DockerCommonRunArgs_Debian8) $(PB_GitDirectory)/Tools/msbuild.sh $(PB_GitDirectory)/src/pkg/packaging/dir.proj $(AdditionalMSBuildProperties) $(DistroSpecificMSBuildArguments) /p:OutputRid=debian.9-$(PB_TargetArchitecture)",
```

Comment:
Build Debian 9 installer using Debian 8 docker container. Starting 2.1.0 , we have moved to produce all deb packages from single ubuntu14.04 docker container.